Library/Formula/phantomjs.rb

```diff
@@ -17,9 +17,13 @@ class Phantomjs < Formula
   depends_on "openssl"
 
   def install
-    inreplace "src/qt/preconfig.sh", "QT_CFG+=' -openssl -openssl-linked'",
-              "QT_CFG+=' -openssl -openssl-linked -I #{Formula["openssl"].opt_include} -L #{Formula["openssl"].opt_lib}'"
-    system "./build.sh", "--confirm", "--jobs", ENV.make_jobs
+    if build.stable?
+      system "./build.sh", "--confirm", "--jobs", ENV.make_jobs,
+             "--qt-config", "-I #{Formula["openssl"].opt_include} -L #{Formula["openssl"].opt_lib}"
+    else
+      inreplace "build.py", "/usr/local", HOMEBREW_PREFIX
```

Comment:
Oh, one more thing: I'd consider rebasing your current history of these branches onto an existing, upstream Qt branch. Ariya's initial "import everything as one commit" makes diffing very difficult as there are no commits in common: https://github.com/qtproject/qtbase/compare/dev...Vitallium:phantomjs